target/xilinx/constraints/vcu118.xdc

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Nicole Narr <narrn@student.ethz.ch>
+# Christopher Reinwardt <creinwar@student.ethz.ch>
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+# Paul Scheffler <paulsc@iis.ee.ethz.ch>
+# Yvan Tortorella <yvan.tortorella@fondazione-chipsit.it>
+
+#############
+# Sys Clock #
+#############
+
+# 250 MHz input clock
+set SYS_TCK 4
```

Comment:
Is there a reason we constrain this differently than the VCU128? We should keep these boards aligned wherever possible, so if both can run faster, they both should.

Reply:
As far as I can understand from the boards DOC, I see that the DDR4 controller on the VCU128 has a fixed 100 MHz fequency clock source, please see the `DDR4 Interface Clock`, page 39 of the user manual. On the contrary, the VCU118 has no constraint on the clock source frequency. The only constraint is on where the sys_clock pin is located on the board, because it must stay in the same IO bank of the DDR4 interface (and as far as I understood the 250 MHz clock connected to the U21 clock buffer, output Q1, is the one which locates in the same IO banks as the DDR interface). In the VCU118 DOC I did not see any 100 MHz clocks to align with the VCU128 unless the programmable clock sources (which I understand need to be programmed by the onboard Zynq 7000).